generator/src/test/java/com/scottlogic/deg/generator/validators/GenerationConfigValidatorTests.java

```diff
@@ -121,13 +143,139 @@ public void fullSequentialWithMaxRowsReturnsValid() {
         );
         testConfigSource.setMaxRows(1234567L);
         GenerationConfig config = new GenerationConfig(testConfigSource);
-        GenerationConfigValidator validator = new GenerationConfigValidator();
+        validator = new GenerationConfigValidator(mockConfigSource, outputTarget, fileUtils);
 
         //Act
-        ValidationResult validationResult = validator.validateCommandLine(config);
+        ValidationResult validationResult = validator.validateCommandLinePreProfile(config);
 
         //Assert
         Assert.assertTrue(validationResult.isValid());
         Assert.assertThat(validationResult.errorMessages, is(empty()));
     }
+
+    @Test
+    public void generateOutputFileAlreadyExists() {
```

Comment:
Just a suggestion for test naming, I tend to go with `methodUnderTest_inputConditions_expectedResult` so this example would be `validateCommandLinePostProfile_withExistingFile_returnsFalse`.